opensearch/src/test/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactoryTest.java

```diff
@@ -763,20 +824,31 @@ public void constructBinary() {
   }
 
   /**
-   * Return the first element if is OpenSearch Array.
+   * Return the all elements if is OpenSearch Array.
    * https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html.
    */
   @Test
-  public void constructFromOpenSearchArrayReturnFirstElement() {
-    assertEquals(integerValue(1), tupleValue("{\"intV\":[1, 2, 3]}").get("intV"));
+  public void constructFromOpenSearchArrayReturnAll() {
```

Comment:
we should also verify field_type_tolerance=false works.

Reply:
Added some tests for this.